src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,148 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+import { Loader } from '../Loader';
+import { ErrorLoadPage } from '../ErrorLoadPage';
+import { PersoneLink } from '../PersoneLink';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  const { slug } = useParams();
+
+  const getDataFromServer = async () => {
+    setIsLoading(true);
+    setIsError(false);
+
+    try {
+      const peopleFromServer = await getPeople();
+
+      const personWithParents = peopleFromServer
+        .map(person => {
+          const mother = peopleFromServer
+            .find(parent => parent.name === person.motherName);
+          const father = peopleFromServer
+            .find(parent => parent.name === person.fatherName);
```

Comment:
Pay attention that find method can return undefined.
Would be great to set the reserve value here

Reply:
Thanks, it's ok, I used it when I was defining what to render in the table. As for me, there is no reserve value, only existing object or false value